### PR TITLE
Fix scroll sideways

### PR DESCRIPTION
**Pull request by casassg**
Resize title of Sponsors and Partners in mobile so we avoid scroll sideways in mobile

### CODE REVIEW

**Comment by mvdan** [comment]
This diff is incredibly large. Looks like you used some kind of IDE that reindented the entire code.

**Comment by Dirbaio** [comment]
indentas con 2 espacios
WHY

**Comment by mvdan** [comment]
Tabs FTW ;)

No but seriously keep the current indentation, always

**Comment by casassg** [comment]
actually I used atom :( Don't know what happened here

**Comment by casassg** [comment]
Fixed

**Comment by mvdan** [comment]
Why move the text? Also makes the diff larger than needed.

**Comment by dasilvacontin** [comment]
_will @mvdan senpai approve my PR ?_
_/me le rolls :8ball:_

**Comment by casassg** [comment]
why you don't accept me ? I moved the text because it needs to override the property when less than 600px and it was before the properties to override 

tl;dr; I moved it because css stuff made me do this

File: src/styles/main.scss [hunk 1]
```diff
@@ -153,6 +142,20 @@ $bwhite: #FFF;
...
+    .head-logo img {
+        width: 85%;
+    }
+    .button-container button {
+        display: block;
+        margin-bottom: 10px;
+        margin-right: 0!important;
+    }
+    .sponsors-title, .partners-title {
+      font-size: 4em;
```
Comment (Dirbaio) [inline src/styles/main.scss]: bad indent :trollface:

Comment (casassg) [inline src/styles/main.scss]: didn't see this here heh

**Comment by Dirbaio** [comment]
:heart:
